pagination/pagetoken.go

```diff
@@ -52,6 +52,11 @@ func ParsePageToken(request Request) (_ PageToken, err error) {
 // Next returns the next page token for the provided Request.
 func (p PageToken) Next(request Request) PageToken {
 	p.Offset += int64(request.GetPageSize())
+
+	if s, ok := request.(skipRequest); ok {
```

Comment:
you can inline this interface if it's only used here and nowhere else:

```go
if skipRequest, ok := request.(interface{
  GetSkip() int32
}); ok {
  p.Offset += int64(skipRequest.GetSkip())
}
```

Reply:
It's being used here as well https://github.com/einride/aip-go/blob/master/pagination/request.go#L35-L37

Reply:
ah nice, then ignore me!